core/templates/pages/contributor-dashboard-page/modal-templates/translation-modal.component.html

```diff
@@ -114,7 +114,7 @@
         Paragraphs are not allowed to be copied. Please make sure to translate the text.
       </p>
       <p *ngIf="hasIncompleteTranslationError">
-        Translation is incomplete. Please make sure all elements in the original content, e.g. images, are included in the translated content.
+  Translation is incomplete. Please use the Copy Tool to include any images or concept cards in the translated content.
```

Comment:
Please fix the indentation so that it matches the structure of the rest of the file.

Reply:
> Please fix the indentation so that it matches the structure of the rest of the file.

@seanlip done, PTAL!